test/unit_tests/braket/circuits/test_circuit.py

```diff
@@ -891,6 +891,35 @@ def test_from_ir_round_trip_transformation():
     assert circuit.to_ir("OPENQASM") == Circuit.from_ir(ir).to_ir("OPENQASM")
 
 
+def test_from_ir_round_trip_transformation_with_targeted_measurements():
+    circuit = (
+        Circuit()
+        .h(0)
+        .cnot(0, 1)
+        .add_instruction(Instruction(Measure(index=2), 1))
+        .add_instruction(Instruction(Measure(index=1), 2))
+        .add_instruction(Instruction(Measure(index=0), 0))
+    )
+    ir = OpenQasmProgram(
+        source="\n".join(
+            [
+                "OPENQASM 3.0;",
+                "bit[3] b;",
+                "qubit[3] q;",
+                "h q[0];",
+                "cnot q[0], q[1];",
+                "b[2] = measure q[1];",
+                "b[1] = measure q[2];",
+                "b[0] = measure q[0];",
+            ]
+        ),
+        inputs={},
+    )
+
+    assert Circuit.from_ir(ir) == Circuit.from_ir(circuit.to_ir("OPENQASM"))
+    assert circuit.to_ir("OPENQASM") == Circuit.from_ir(ir).to_ir("OPENQASM")
+
+
```

Comment:
How to get around breaking dependencies:
1. Remove this test from this PR and revert the versions in `setup.py` and `tox.ini`
2. Merge this PR
3. Merge https://github.com/amazon-braket/amazon-braket-default-simulator-python/pull/266
4. Create and merge new PR that reintroduces this test and bumps the default simulator version in `setup.py` to latest release 
```suggestion
```

Reply:
I've removed the test and reverted the targets for the default simulator version in the setup.py and tox.ini files in the following commit: [7b3ebf3](https://github.com/amazon-braket/amazon-braket-sdk-python/pull/1006/commits/7b3ebf3ffab8e442e0c87d60c003d46cc664b227) !